backend/applications/api/v1/views.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import mixins, viewsets
+
+from applications.api.v1.serializers import ApplicationSerializer
+from applications.models import Application
+
+
+class ApplicationViewSet(
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    viewsets.GenericViewSet,
+):
+    queryset = Application.objects.all()
```

Comment:
This will probably need prefetching sooner than later, something like `Application.objects.select_related("company").prefetch_related("summer_bills")`